src/css/landing-page.css

```diff
@@ -217,7 +217,7 @@
 
 .card-row .column .content {
   margin-bottom: 16px;
-  display: inline-block;
+  /* display: inline-block; */
```

Comment:
what effect does deleting this have?
I could imagine inline-block allowing cards to display horizontally?